### PR TITLE
[`RWKV`] Add note in doc on `RwkvStoppingCriteria`

### PR DESCRIPTION
**Pull request by ArthurZucker**
# What does this PR do?
Given the usage of RWKV, decide to go with a tip in the doc rather than changing the `generate` and adding `RwkvStoppingCriteria` to the library. Adresses #23852

### CODE REVIEW

File: docs/source/en/model_doc/rwkv.md [hunk 1]
```diff
@@ -51,6 +51,20 @@ output_two = outputs.last_hidden_state
 torch.allclose(torch.cat([output_one, output_two], dim=1), output_whole, atol=1e-5)
 ```
 
+If you want to make sure the model stops generating when `'\n\n'` is detected, we recommend using the following stopping criteria:
+
+```python
```
Comment (younesbelkada) [inline docs/source/en/model_doc/rwkv.md:56]:
```suggestion
```python 
import torch
```

**Review by younesbelkada — COMMENTED** [review]
Thanks for working on this, left a nit :D

File: docs/source/en/model_doc/rwkv.md (hunk 1 above)
Comment (younesbelkada) [reply docs/source/en/model_doc/rwkv.md:56]: ah yes, makes sense !

**Review by sgugger — APPROVED** [review]
Thanks!

Can you just format the code with new lines ;-)

**Pull request merged**